cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1110,6 +1110,12 @@
     public static final String CFG_CDAP_MASTER_KRB_PRINCIPAL = "cdap.master.kerberos.principal";
 
     public static final String UGI_CACHE_EXPIRATION_MS = "cdap.ugi.cache.expiration.ms";
+    /** Case conversion type */
+    public static final String SECURITY_AUTH_USERNAME_CASE_CONVERSION_TYPE =
+            "security.authentication.handler.usernameCaseConversion";
```

Comment:
Please use `security.auth.server.` as the configuration prefix to be consistent with existing configurations

Reply:
done